data-raw/watershed/antelope/antelope_creek.Rmd

```diff
@@ -16,35 +16,30 @@ library(cvpiaHabitat)
 ## Future Data Improvements
 None Planned 
 
-## Instream Spawning and Rearing Habiat
-```{r}
-source('data-raw/watershed/regional_approximation/mean_wua_description.R')
-```
-**Data Source:** 
-
-`r print_mean_wua_details('Antelope Creek')`
+## Instream Spawning and Rearing Habitat
 
-### Spawning and Rearing Modeling Details 
+**Data Source:** [Calculated with a relationship between flow and mean weighted usable area from watersheds in the region with modeled habitat](http://cvpia-habitat-docs-markdown.s3-website-us-west-2.amazonaws.com/watershed/Regional_Approximation.html)
+```{r, include=FALSE}
+source('data-raw/watershed/regional_approximation/description_regional_approx.R')
+```
 
-The spawning and rearing data used for Antelope Creek is the regional approximation for the Upper-mid Sacramento River Region. More information about this methodology can be found [here](http://cvpia-habitat-docs-markdown.s3-website-us-west-2.amazonaws.com/watershed/Regional_Approximation.html).  
+`r print_regional_approx('Antelope Creek')`
 
 ```{r, include=FALSE}
-source('data-raw/floodplain/floodplain_utils.R')
+source('data-raw/watershed/floodplain/floodplain_utils.R')
 ```
 
-## Floodplain Modeling Details
-
-**Data Source:**
+## Floodplain Rearing Habitat 
 
-[Scaled from a Deer Creek flow to floodplain area relationship generated with a 1D HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DRAFT+Deer+Creek+Hydraulic+Models+Tech+Memo+6-08-07.pdf)
+**Data Source:** [Scaled from a Deer Creek flow to floodplain area relationship generated with a 1D HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DRAFT+Deer+Creek+Hydraulic+Models+Tech+Memo+6-08-07.pdf)
 
 `r print_model_details('Antelope Creek', 'fr')`
```

Comment:
make sure to rerun this file after fixing the issues i flagged in the `print_model_details`

Reply:
re-ran